harvard-university-for-the-creative-arts.csl

```diff
@@ -191,13 +227,14 @@
           <text term="in" text-case="capitalize-first" suffix=": "/>
           <names variable="editor" delimiter="." suffix=" ">
             <name and="text" delimiter-precedes-last="never" et-al-min="3" et-al-use-first="1" initialize-with="." name-as-sort-order="all"/>
-            <label form="short" plural="never" prefix=" (" suffix=")"/>
+            <label form="short" plural="never" text-case="lowercase" prefix=" (" suffix=")"/>
             <et-al font-style="italic"/>
           </names>
           <group delimiter=", ">
             <group font-style="normal" font-variant="normal">
               <text variable="container-title" font-style="italic" suffix="."/>
               <text variable="collection-title" font-style="italic"/>
+              <text variable="volume" prefix=" Vol " suffix="."/>
```

Comment:
@ibadger, you can use the short form of the "volume" term here (see https://github.com/citation-style-language/locales/blob/96042867e58040158417e457e3cb78c73d36b029/locales-en-GB.xml#L227), with a new group to print the term conditionally. Let me know if you need help with this.